.github/workflows/release-automated.yml

```diff
@@ -26,13 +26,13 @@ jobs:
       - name: Setup dart
         uses: dart-lang/setup-dart@v1
         with:
-          sdk: '3.0'
+          sdk: '3.2.6'
       - name: Setup flutter
         if: env.package == 'flutter'
         uses: subosito/flutter-action@v2
         with:
-          channel: 'stable'
-          cache: true
```

Comment:
Why was cache: true removed?

Reply:
The reason for using cache was not clear to me.
Is there a specific reason?

Reply:
It speeds up the CI